pytorch_lightning/accelerators/accelerator.py

```diff
@@ -198,71 +201,64 @@ def training_step(
                 - hiddens(:class:`~torch.Tensor`): Passed in if
                   :paramref:`~pytorch_lightning.core.lightning.LightningModule.truncated_bptt_steps` > 0.
         """
-        args[0] = self.to_device(args[0])
+        step_kwargs = self.to_device(step_kwargs)
 
         with self.precision_plugin.train_step_context(), self.training_type_plugin.train_step_context():
-            return self.training_type_plugin.training_step(*args)
+            return self.training_type_plugin.training_step(*step_kwargs.values())
```

Comment:
another n00b question: with dict the order can shuggle also as dist the nb items may yield in order variation...

Reply:
ok, found the `_build_kwargs` but still just thinking if use may accidentally work with this API and mess with passed arguments...

Reply:
sorry I didn't get you. We are using ordered dict so there won't be an issue with order variation.